types/d3-polygon/d3-polygon-tests.ts

```diff
@@ -18,6 +18,7 @@ let point: [number, number] = [15, 15];
 const polygon: Array<[number, number]> = [[10, 10], [20, 20], [10, 30]];
 const pointArray: Array<[number, number]> = [[10, 10], [20, 20], [10, 30], [15, 15]];
 let hull: Array<[number, number]>;
+let hullOrNoting: Array<[number, number]> | null;
```

Comment:
Nitpick: The "nothing" in `hullOrNoting` is misspelled.

Reply:
👍